develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/06-liferay-util-html-bottom.markdown

```diff
@@ -0,0 +1,28 @@
+# Using Liferay Util HTML Bottom [](id=using-liferay-util-html-bottom)
+
+The HTML bottom tag is not a self-closing tag. Content placed between the 
+opening and closing of this tag is moved to the bottom of the `<html>` tag. When 
+something is passed using this taglib, the 
+[bottom.jsp](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/common/themes/bottom.jsp#L53-L59) 
+is passed markup and outputs in this JSP. 
+
+The example configuration below uses the `<liferay-util:html-bottom>` tag to 
+include JavaScript (a common use case) provided by the portlet's bundle:
+    
+    <liferay-util:html-bottom outputKey="addscript">
```

Comment:
Shouldn't `outputKey` be `htmlbottom` for consistency with the others?

Reply:
👍